yarn.config.cjs

```diff
@@ -1,32 +1,328 @@
+// @ts-check
```

Comment:
Aligns with core but has been modified to be more generic when handling critical dependencies